docs/topics/native/native-get-started.md

```diff
@@ -23,22 +23,43 @@ To get started, install the latest version of [IntelliJ IDEA](https://www.jetbra
 
 Your project will open. By default, the wizard creates the necessary `main.kt` file with code that prints "Hello, Kotlin/Native!" to the standard output.
 
-The `build.gradle.kts` file contains the project settings. Read more about these settings in the [Kotlin Multiplatform Gradle DSL reference](mpp-dsl-reference.md).
+The `build.gradle.kts` file contains the project settings. Make sure you use the latest version of the Kotlin Multiplatform
+Gradle plugin:
 
-## Run the application
+```kotlin
+plugins {
+    kotlin("multiplatform") version "%kotlinVersion%"
+}
+```
+   
+Read more about these settings in the [Kotlin Multiplatform Gradle DSL reference](mpp-dsl-reference.md).
 
-Start the application by clicking **Run** next to the run configuration at the top of the screen.
+## Build and run the application
 
-![Run the application](native-run-app.png){width=600}
+1. Click **Build Project** next to the run configuration at the top of the screen:
 
-IntelliJ IDEA opens the **Run** tab and shows the output:
-![Application output](native-output-1.png){width=600}
+   ![Build the application](native-run-app.png){width=600}
+
+2. On the **Terminal** tab, run the following command:
+
+   ```bash
+   build/bin/native/debugExecutable/<your_app_name>.kexe
+   ```
+
+   IntelliJ IDEA prints "Hello, Kotlin/Native!".
```

Comment:
Not sure I understand you correctly. I don't think there's a need to add a screenshot for this output. It just prints out one line. Backticks are for the command, so that you can copy it to clipboard